tracer/src/Datadog.Trace/Ci/Configuration/CIVisibilitySettings.cs

```diff
@@ -231,7 +231,7 @@ private TracerSettings InitializeTracerSettings()
                                       },
                                       ConfigurationOrigins.Calculated));
 
-            var tracerSettings = new TracerSettings(source, new ConfigurationTelemetry());
+            var tracerSettings = new TracerSettings(source, new ConfigurationTelemetry(), new());
```

Comment:
For readability, can we add either the name of the new parameter here (`errorLog: new()`) or the name of the type we are instantiating (`new OverrideErrorLog()`?

(Personally, I'd prefer the type name. I don't think using `new()` everywhere where it is valid is helping anyone.)

Reply:
Yeah, fair enough, I mostly used `new()` where we don't _care_ what it is, but that only really applies in tests, definitely not here!